reference/format/datagrid.rst

```diff
@@ -0,0 +1,166 @@
+Datagrid
+========
++-----------+------------------+
+| Filename  | ``datagrid.yml`` |
++-----------+------------------+
+| Root Node | ``datagrid``     |
++-----------+------------------+
+
+The datagrid configuration is a large file that is a map of datagrid names mapped to options that
+configure their behavior:
+
+.. _reference-datagrid-columns:
+
+``actions`` (**type**: ``map``)
+
+    This a map of actions that can be performed from the datagrid. The keys are used to give unique
+    identifiers inside the grid, but do not have a special meaning. For each action, a map must be
+    passed configuring the action using the following options:
+
+    ``icon`` (**type**: ``string``)
+
+        The name of the Font Awesome icon that represents the action.
+
+    ``label`` (**type**: ``string``)
+
+        The action label that is displayed when the user hovers the icon. This will automatically
+        be translated.
+
+    ``link`` (**type**: ``string``)
+
+        The name of a link to associate with this action which is configured in the ``properties``
+        section.
+
+    ``rowAction`` (**type**: ``boolean`` **default**: ``false``)
+
+        When set to ``true``, the user can click anywhere on the row to perform this action.
+
+    ``type`` (**type**: ``string``)
+
+        This can either be ``navigate`` or ``delete``. When the value is ``delete``, an HTTP DELETE
```

Comment:
There are more types than these two, see https://github.com/laboro/platform/blob/f5bee512c2ddae2b13b1d1de1da6c2af921861ac/src/Oro/Bundle/DataGridBundle/Resources/config/actions.yml. I'd propose to tell that these two are most commonly used.